test/test_zipper_boundary_conditions.jl

```diff
@@ -0,0 +1,43 @@
+include("dependencies_for_runtests.jl")
+
+using OrthogonalSphericalShellGrids: Zipper
+
+@testset "Zipper boundary conditions..." begin
+    grid = TripolarGrid(size = (10, 10, 1))
+    Nx, Ny, _ = size(grid)
+    Hx, Hy, _ = halo_size(grid)
+
+    c = CenterField(grid)
+    u = XFaceField(grid)
+    v = YFaceField(grid)
+
+    @test c.boundary_conditions.north.classification isa Zipper
+    @test u.boundary_conditions.north.classification isa Zipper
+    @test v.boundary_conditions.north.classification isa Zipper
+
+    @test c.boundary_conditions.north.condition == 1
+    @test u.boundary_conditions.north.condition == -1
+    @test v.boundary_conditions.north.condition == -1
+
+    set!(c, 1.0)
+    set!(u, 1.0)
+    set!(v, 1.0)
+
+    fill_halo_regions!(c)
+    fill_halo_regions!(u)   
+    fill_halo_regions!(v)
+
+    north_boundary_c = view(c.data, :, Ny+1:Ny+Hy, 1)
+    north_boundary_v = view(v.data, :, Ny+1:Ny+Hy, 1)
+    @test all(north_boundary_c .== 1.0)
+    @test all(north_boundary_v .== -1.0)
```

Comment:
```suggestion
    @test all(north_boundary_c .== 1)
    @test all(north_boundary_v .== -1)
```

Try to remember not to hard code Float64. Here, we will want to extend this test to Float32 eventually. The code above would fail.